@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -881,11 +892,15 @@ describe("MapView", function() {
         for (let x = -100; x <= 100; x += 100) {
             for (let y = -100; y <= 100; y += 100) {
                 mapView = new MapView({ canvas: (customCanvas as any) as HTMLCanvasElement });
+                await mapView.getTheme();
+                await mapView.sceneEnvironment.backgroundDataSourcePromise;
```

Comment:
Disable the background data source instead, it's not needed for the test. Pass "addBackgroundDataSource: false" to the map view options.

Reply:
I had done this, but thought it was needed for the test below, so thought to keep it in. I will go back to it....

However, it is a bit suspicious that a test needs to disable the background data source to work. I will make a ticket to propose to fix this.